src/main/java/sirius/biz/tycho/kb/SynchronizeArticlesTask.java

```diff
@@ -93,7 +97,10 @@ private void synchronizeArticles() {
         String syncId = keyGenerator.generateId();
         Sirius.getClasspath()
               .find(Pattern.compile("(default/|customizations/[^/]+/)?kb/.*\\.pasta"))
-              .forEach(matcher -> updateArticle(cleanupTemplatePath(matcher.group(0)), syncId));
+              .map(matcher -> cleanupTemplatePath(matcher.group(0)))
+              .filter(templatePath -> !(isTemplateInsidePartsDirectory(templatePath) || isTemplateAPartOfAnArticle(
+                      templatePath)))
```

Comment:
```suggestion
              .filter(templatePath -> !isTemplateInsidePartsDirectory(templatePath))
              .filter(templatePath -> !isTemplateAPartOfAnArticle(templatePath))
```
Code-style: Simplify Filters